typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -943,6 +943,67 @@ export const polygonzkevm: ChainMetadata = {
   },
 };
 
+export const neutrontestnet: ChainMetadata = {
+  protocol: ProtocolType.CosmWasm,
+  domainId: 33333,
+  chainId: 33333,
+  name: Chains.neutrontestnet,
+  displayName: 'Neutron Testnet',
+  displayNameShort: 'Neutron Testnet',
+  nativeToken: {
+    name: 'Neutron',
+    symbol: 'NTRN',
+    decimals: 6,
+  },
+  blocks: {
+    confirmations: 1,
+    reorgPeriod: 0,
+    estimateBlockTime: 3,
+  },
+  rpcUrls: [{ http: 'http://54.149.31.83:26657' }],
```

Comment:
To get the UI's keplr integration to work, I needed **both** an RPC url and a REST url (cosmos has 3 types, RPC, gRPC, and REST). We'll need a rest api here too, which I believe is `http://54.149.31.83:1317`. For simplicity, the warp UI assumes the 1st is RPC and the 2nd is REST for cosmos chains